test/object-storage.test.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+var assert = require('assert');
+
+describe('object-storage connector', function() {
+  it('should be set up as loopback-component-storage', function(done) {
+    this.timeout(10000);
+    var app = require('./fixtures/object-storage-app');
+    var booted = false;
+    app.on('booted', function() {
+      if (booted) return;
```

Comment:
First of all, I find it rather weird that `booted` is emitted more than once.

Let's not worry about that though. If you want your event listener to be called only once, then you should use `once` instead of `on`. That way you don't need to maintain the flag `booted`.

```js
app.once('booted', () => {
  // ...
});
```